### DISCUSSION TITLE
Hardware-Back-Button with Uno.Extensions

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno.extensions#647] [Q&A]
How is handling of the Hardware-Back-Button supposed to be done with Uno.Extensions?

**Comment by nickrandolph** [comment]
@TopperDEL currently there isn't any specific support in uno extensions. There is a GoBack method on the INavigator which will locate the deepest frame in the navigation hierarchy and attempt to go back on it - useful if you are intercepting back button and want to navigate back to a previous page

**Reply by TopperDEL** [reply]
What would be the way to go then to get the HardwareBackButton used?
